app/src/main/java/org/wikipedia/page/linkpreview/LinkPreviewDialog.kt

```diff
@@ -145,12 +147,25 @@ class LinkPreviewDialog : ExtendedBottomSheetDialogFragment(), LinkPreviewErrorV
         }
     }
 
+    private val requestEditSectionLauncher = registerForActivityResult(ActivityResultContracts.StartActivityForResult()) {
+        if (it.resultCode == EditHandler.RESULT_REFRESH_PAGE) {
+            FeedbackUtil.showMessage(this, R.string.edit_saved_successfully)
```

Comment:
For showing the edit-success message, the context should be the parent activity (`requireActivity()`) instead of `this`, and then the dialog can be dismissed right after.
This way we don't need to worry about reloading the stub.